Resources/Prototypes/Recipes/Construction/machines.yml

```diff
@@ -5,6 +5,7 @@
   startNode: start
   targetNode: computer
   category: construction-category-machines
+  description: A computer with a built-in screen.
```

Comment:
for this and the machine frame, their descriptions will have to be special, since the graphs for both of them encompass a wide array of different entities (depending on what circuitboard is used to construct it). so they should probably read something like `A frame used to construct anything with a [machine/computer] circuitboard.`